.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
+- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
+- [ ] Gallery example in `./doc/examples` (new features only)
+- [ ] Unit tests
+
+[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]
+
+## References
```

Comment:
OK, how about:

- If this is a bug-fix or enhancement, it addresses issue #

Again, it's not the end of the world if we don't get *all* of the information we need.  At worst, it is exactly the situation we had before.